apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/inline-support-link.tsx

```diff
@@ -22,16 +33,19 @@ export default function DescriptionSupportLink( { children, title, url }: Props
 			{ children }
 			<br />
 			<ExternalLink
-				onClick={ () => {
+				onClick={ ( e ) => {
+					e.preventDefault();
 					recordTracksEvent( 'calypso_block_description_support_link_click', {
 						block: title,
 						support_link: url,
 					} );
+					setShowHelpCenter( true );
+					setShowSupportDoc( localizeUrl( url ), postID );
 				} }
 				ref={ ( reference ) => ref !== reference && setRef( reference ) }
 				style={ { display: 'block', marginTop: 10, maxWidth: 'fit-content' } }
 				className="fse-inline-support-link"
-				href={ url }
+				href="#"
```

Comment:
Since we're preventing default above, we should keep this url, in case someone wants to read in a new tab.

Reply:
Reverted this change